a11y.js

```diff
@@ -29,6 +29,7 @@ const pageTypes = {
     '/iframe', // known issue above with iframes should be revisited once https://github.com/bbc/bbc-a11y/pull/313 gets merged.
     "//div[@id='root']/div/div[1]/main/div[37]/div/div/div", // issue with IDT2 includes
   ],
+  idxPage: ["//div[@id='root']/header/nav/div/div[1]/div/ul"],
```

Comment:
I was wondering whether we could rename `idxPage` to `indexPage` (just for clarity and consistency with other page types e.g. MAP = Media Asset Page, PGL = Photo Gallery Page etc).

Reply:
That's a good point, but it makes more sense for me to keep it as `idxPage` to match with the rest of the IDX codebase, like for example the route file `https://github.com/bbc/simorgh/blob/latest/src/app/routes/idx/index.js` or the page type `https://github.com/bbc/simorgh/tree/latest/src/app/pages/IdxPage` where call it `idx`

Reply:
This was a somewhat unfortunate decision we had to make early on since if we stuck with the naming convention of spelling it out like with other page types, sometimes it ended up just being `../index` which caused some issues with certain paths to `./index.js` files. We ended up deciding it would be a better trade off if we just used IDX where we could.

Reply:
Thanks for the clarification.